docs/overview.rst

```diff
@@ -430,6 +430,21 @@ TMT_PLAN_DATA
     back from the guest and available for inspection after the
     plan is completed.
 
+TMT_PLAN_ENVIRONMENT_FILE
+    Path to the file containing environment variables that should
+    be sourced after prepare and execute steps. These variables will
+    then be accessible for all subsequent steps. Variables inside
+    the file has to be in the format of ``NAME=VALUE`` and each
+    variable should be on a separate line.
+
+    Example of the file content::
+
+        COUNT=1
+        VARIABLE=VALUE
+        ANOTHER_VARIABLE=ANOTHER_VALUE
+
+.. versionadded:: 1.29
```

Comment:
I think this should be indented to the level of the text above, to be rendered for the text above.

Reply:
Fixed in 8ec6eee.